src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -1933,7 +1933,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' Symbol for the type or null if type cannot be found or is ambiguous. 
         ''' </returns>
         Friend Shadows Function GetTypeByMetadataName(fullyQualifiedMetadataName As String) As NamedTypeSymbol
-            Return Me.Assembly.GetTypeByMetadataName(fullyQualifiedMetadataName, includeReferences:=True, isWellKnownType:=False, conflicts:=Nothing)
+            Return Me.Assembly.GetTypeByMetadataName(fullyQualifiedMetadataName, includeReferences:=True, isWellKnownType:=False, conflicts:=Nothing,
+                                                     ignoreCorLibraryDuplicatedTypes:=Me.Options.IgnoreCorLibraryDuplicatedTypes)
```

Comment:
>ignoreCorLibraryDuplicatedTypes [](start = 53, length = 31)

:memo: This is to bring `GetWellKnownType` and `GetTypeByMetadataName` closer in alignment (they still disagree when dealing with types that fail the `IsValidWellKnownType` test)

Reply:
> This is to bring GetWellKnownType and GetTypeByMetadataName closer in alignment (they still disagree when dealing with types that fail the IsValidWellKnownType test)
Same change was made in C# side

I don't think we should make this change. The option is internal and only used by EE, I believe.

---
In reply to: [585942375](https://github.com/dotnet/roslyn/pull/51571#discussion_r585942375) [](ancestors = 585942375)